src/main/java/com/bina/varsim/util/StringUtilities.java

```diff
@@ -51,10 +52,39 @@ public static String concatenateArray(final ChrString[] a, final CharSequence de
         if (a == null) {
             return "";
         }
-        StringJoiner joiner = new StringJoiner(delimiter);
+        StringJoiner joiner = new StringJoiner(delimiter.toString());
         for (int i = 0; i < a.length; i++) {
             joiner.add(a[i].toString());
         }
         return joiner.toString();
     }
+
+    /**
+     * partial implementation of Java 8 StringJoiner
+     */
+    public static class StringJoiner {
```

Comment:
Oh, I saw that on VarSim manual, java 7 was required. Shall we change it to Java 8?